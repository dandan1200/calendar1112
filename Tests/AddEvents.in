python calendar.py ADD 10-10-2000 "Birthday Party" "At Home"
python calendar.py GET DATE 10-10-2000
python calendar.py ADD 10-10-2000 "Meeting" "In park"
python calendar.py GET DATE 10-10-2000

