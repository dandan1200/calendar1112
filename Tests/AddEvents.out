10-10-2000 : Birthday Party : At Home
10-10-2000 : Birthday Party : At Home
10-10-2000 : Meeting : In park
